tests/fsharp/core/printing/output.stderr.txt

```diff
@@ -0,0 +1,348 @@
+
```

Comment:
None of these *.txt should be added in this PR, they should be added to `.gitignore`